src/jit/morph.cpp

```diff
@@ -1367,7 +1367,7 @@ void fgArgInfo::RemorphStkArg(
     nextSlotNum += numSlots;
 }
 
-void fgArgInfo::SplitArg(unsigned argNum, unsigned numRegs, unsigned numSlots)
+void fgArgInfo::SplitArg(unsigned argNum, unsigned numRegs, unsigned numSlots, bool isReMorph)
```

Comment:
@CarolEidt `RemorphStkArg` also updates `nextSlotNum`. But in remorphing, `nextSlotNum` seems to be used for assertion check only. IMO, we can modify to skip updating `nextSlotNum` using `DEBUG` flag and `reMorphing` check.